docs/index.rst

```diff
@@ -0,0 +1,38 @@
+.. meta::
+    :description: HSA runtime implementation
+    :keywords: ROCR, ROCm, library, tool, runtime
+
+.. _index:
+
+=====================
+ROCR documentation
+=====================
+
+The ROCm runtime (ROCR) is AMD's implementation of HSA runtime, which is a thin, user-mode API that exposes the necessary interfaces to access and interact with graphics hardware driven by the AMDGPU driver set and the ROCK kernel driver. To learn more, see :ref:`what-is-rocr-runtime`
```

Comment:
this link is broken:
![image](https://github.com/ROCm/ROCR-Runtime/assets/164940351/03a39a6a-2bf0-40f4-b591-7a23fb87e0a1)

Reply:
corrected.